src/index.js

```diff
@@ -1,81 +1,13 @@
-const bunyan = require('bunyan');
-const config = require('./config');
-const pkg = require('./package');
-const _ = require('lodash');
-
-const bunyanStream = {
-  stream: process.stdout
-};
-
-const mainLogger = bunyan.createLogger({
-  name: pkg.name,
-  version: pkg.version,
-  level: config.logLevel || 'info',
-  serializers: {
-    error: bunyan.stdSerializers.err
-  },
-  streams: [bunyanStream]
+const Logger = require('./libs/Logger');
+const createUdpStream = require('./helpers/createUdpStream');
+const env = process.env;
+const destination = env.LOG_ENDPOINT
+  ? createUdpStream(env.LOG_ENDPOINT)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Convert this usage of the ternary operator to an "if"/"else" structure. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AConditionalOperator)